frontend/src/metabase/ui/components/data-display/Card/Card.styled.tsx

```diff
@@ -0,0 +1,18 @@
+import type { MantineTheme, MantineThemeOverride } from "@mantine/core";
+
+export const getCardOverrides = (): MantineThemeOverride["components"] => ({
```

Comment:
How do you determine what style should be overridden?

Reply:
Initially we thought that no styles need to be changed. But by looking at the implementation they use custom border colors that we need to override.